Config/Container.php

```diff
@@ -694,15 +694,15 @@ function toArray($useAttr = true)
         $array[$this->name] = array();
         switch ($this->type) {
             case 'directive':
-                if ($useAttr && count($this->attributes) > 0) {
+                if ($useAttr && is_countable($this->attributes) && count($this->attributes) > 0) {
                     $array[$this->name]['#'] = $this->content;
                     $array[$this->name]['@'] = $this->attributes;
                 } else {
                     $array[$this->name] = $this->content;
                 }
                 break;
             case 'section':
-                if ($useAttr && count($this->attributes) > 0) {
+                if ($useAttr && is_countable($this->attributes) && count($this->attributes) > 0) {
```

Comment:
Would change it to a `if ($useAttr && !empty($this->attributes)) {` maybe?

Reply:
Thank you, much cleaner this way.